vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -1,10 +1,31 @@
 package container
 
 import (
+	"time"
 	"github.com/docker/engine-api/types/strslice"
 	"github.com/docker/go-connections/nat"
 )
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+type HealthConfig struct {
+	// Test to perform to check container is healthy.
+	// An empty slice means to inherit the default.
+	// The options are:
+	// {} : inherit healthcheck
+	// {"NONE"} : disable healthcheck
+	// {"CMD", args...} : exec arguments directly
+	// {"CMD-SHELL", args...} : run command with system's default shell
+	Test strslice.StrSlice `json:",omitempty"`
```

Comment:
Glad I just caught this: `strslice.StrSlice` is for backwards compatibility. Just use `[]string`. There is not need to support string commands for new stuff.